### PR TITLE
Event inspector was setting the start date to current date +1 week an…

### PR DESCRIPTION
**Pull request by FBRTMaka**
…d +1 month.  Fixed it to display the start date of the event.

@DanielJMaher , @ssontag55 

With the change to the way the events are delivered, and seeing the way wrong start date in the event inspector, these changes populate the Start/End date of the event correctly and are pre formatted.

### CODE REVIEW

File: ooiui/static/js/views/asset_management/EventViewPage.js [hunk 1]
```diff
@@ -187,24 +169,6 @@ var EventViewPage = Backbone.View.extend({
...
-        });
-        $( "#enddate_d" ).datepicker({
-          changeMonth: true,
-          numberOfMonths: 2,
-          maxDate: "+1m +1w",
-          onClose: function( selectedDate ) {
-            $( "#startdate_d" ).datepicker( "option", "maxDate", selectedDate );
-          }
-        });
-
```
Comment (FBRTMaka) [inline ooiui/static/js/views/asset_management/EventViewPage.js]: @DanielJMaher , @ssontag55 This was causing the date to re render with the current date +1m and +1w, and not the event start date.  SO I removed the date picker in lieu of the correct start/end dates.  Adding the date picker back in would be nice, but for this demo I think it's best not to have conflicting dates.

**Comment by ssontag55** [comment]
@FBRTMaka @DanielJMaher  I can add back in with the right date.  All done on the service side?

**Comment by DanielJMaher** [comment]
@FBRTMaka @ssontag55 Where are we here?

**Comment by FBRTMaka** [comment]
@ssontag55 yeah it's all done in the services now, so no conversions needed.  If you want to add it back in with the date picker that's sweet!

**Comment by ssontag55** [comment]
@FBRTMaka Ok, I've done it, but will merge after this PR

**Pull request merged**